unit-tests/Tests/Version3/functions.php

```diff
@@ -82,7 +82,11 @@ public function test_wc_rest_upload_image_from_url_should_return_error_when_unab
 	 */
 	public function test_wc_rest_upload_image_from_url_should_return_error_when_invalid_image_is_passed() {
 		// empty file.
-		$expected_error_message = 'Invalid image: File is empty. Please upload something more substantial. This error could also be caused by uploads being disabled in your php.ini or by post_max_size being defined as smaller than upload_max_filesize in php.ini.';
+		if ( version_compare( get_bloginfo( 'version' ), '5.4-alpha', '>=' ) ) {
```

Comment:
Instead of checking for the EXACT error, perhaps we can look for `Invalid image: File is empty.` at the start?

Reply:
Makes sense. Added in https://github.com/woocommerce/woocommerce-rest-api/pull/184/commits/3ba539464f5342009e3cadff8f1cd3e098613372